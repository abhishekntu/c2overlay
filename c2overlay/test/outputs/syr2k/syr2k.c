/****************************************************************/
/*   chebyshev		                                        */
/****************************************************************/

#ifdef TEST
#include <stdio.h>
#endif

#include<math.h>

#ifdef ALGO
void foo()
{
    int A [9];
    int B [9];
    int C [9];
    int D [9];
    int alpha = 9;
    int beta = 10;
    int i, j;
    {
        {
            D[0] = (beta * C [0]) +(alpha * (A[0] * B[0] + A[1] * B[1] + A[2] * B[2] + B[0] * A[0] + B[1] * A[1] + B[2] * A[2]));
            D[1] = (beta * C [1]) +(alpha * (A[0] * B[3] + A[1] * B[4] + A[2] * B[5] + B[0] * A[3] + B[1] * A[4] + B[2] * A[5]));
            D[2] = (beta * C [2]) +(alpha * (A[0] * B[6] + A[1] * B[7] + A[2] * B[8] + B[0] * A[6] + B[1] * A[7] + B[2] * A[8]));
        }
        {
            D[3] = (beta * C [3]) +(alpha * (A[3] * B[0] + A[4] * B[1] + A[5] * B[2] + B[3] * A[0] + B[4] * A[1] + B[5] * A[2]));
            D[4] = (beta * C [4]) +(alpha * (A[3] * B[3] + A[4] * B[4] + A[5] * B[5] + B[3] * A[3] + B[4] * A[4] + B[5] * A[5]));
            D[5] = (beta * C [5]) +(alpha * (A[3] * B[6] + A[4] * B[7] + A[5] * B[8] + B[3] * A[6] + B[4] * A[7] + B[5] * A[8]));
        }
        {
            D[6] = (beta * C [6]) +(alpha * (A[6] * B[0] + A[7] * B[1] + A[8] * B[2] + B[6] * A[0] + B[7] * A[1] + B[8] * A[2]));
            D[7] = (beta * C [7]) +(alpha * (A[6] * B[3] + A[7] * B[4] + A[8] * B[5] + B[6] * A[3] + B[7] * A[4] + B[8] * A[5]));
            D[8] = (beta * C [8]) +(alpha * (A[6] * B[6] + A[7] * B[7] + A[8] * B[8] + B[6] * A[6] + B[7] * A[7] + B[8] * A[8]));
        }
    }

}
#endif


#ifdef TEST
int main(void)
{
	int x;
 	int result;
  	foo();
  	return 0;
}
#endif
