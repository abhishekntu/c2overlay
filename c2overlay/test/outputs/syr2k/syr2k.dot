digraph graphname {
N29 [color="black", label="N86N90_mul_Imm_9_N86_add_N90_N29"];
N28 [color="black", label="N29N87_mul_N29_add_N87_N28"];
N23 [color="black", label="load_Imm_8_N18_N23"];
N22 [color="black", label="load_Imm_1_N15_N22"];
N21 [color="black", label="load_Imm_1_N14_N21"];
N20 [color="black", label="load_Imm_5_N17_N20"];
N27 [color="black", label="load_Imm_5_N7_N27"];
N26 [color="black", label="load_Imm_5_N9_N26"];
N25 [color="black", label="load_Imm_1_N8_N25"];
N24 [color="black", label="load_Imm_2_N19_N24"];
N85 [color="black", label="mul_N71_N85"];
N84 [color="black", label="mul_Imm_10_N70_N84"];
N87 [color="black", label="N73N48_mul_Imm_9_N73_add_N48_N87"];
N86 [color="black", label="mul_N72_N86"];
N81 [color="black", label="mul_Imm_10_N123_N81"];
N80 [color="black", label="mul_Imm_10_N120_N80"];
N83 [color="black", label="mul_Imm_10_N76_N83"];
N82 [color="black", label="mul_Imm_10_N75_N82"];
N89 [color="black", label="N41N136_mul_N41_add_N136_N89"];
N88 [color="black", label="N78N101_mul_N78_add_N101_N88"];
N38 [color="black", label="N92N105_mul_N92_add_N105_N38"];
N39 [color="black", label="mul_N93_N39"];
N30 [color="black", label="N81N153_add_N81_add_N153_N30"];
N31 [color="black", label="N89N37_mul_N89_add_N37_N31"];
N32 [color="black", label="N88N77_mul_N88_add_N77_N32"];
N33 [color="black", label="N38N132_mul_N38_add_N132_N33"];
N34 [color="black", label="mul_N39_N34"];
N35 [color="black", label="N33N85_mul_N33_add_N85_N35"];
N36 [color="black", label="N34N149_add_N34_add_N149_N36"];
N37 [color="black", label="N36N150_add_N36_add_N150_N37"];
N98 [color="black", label="mul_N141_N98"];
N99 [color="black", label="mul_Imm_10_N140_N99"];
N92 [color="black", label="mul_N42_N92"];
N93 [color="black", label="mul_N45_N93"];
N90 [color="black", label="mul_N40_N90"];
N91 [color="black", label="N43N80_add_N43_add_N80_N91"];
N96 [color="black", label="mul_N143_N96"];
N97 [color="black", label="N142N79_mul_N142_add_N79_N97"];
N94 [color="black", label="N47N82_mul_N47_add_N82_N94"];
N95 [color="black", label="mul_N46_N95"];
N114 [color="black", label="store_Imm_7_N154_N114"];
N115 [color="black", label="store_Imm_0_N155_N115"];
N116 [color="black", label="store_Imm_2_N156_N116"];
N117 [color="black", label="store_Imm_8_N157_N117"];
N110 [color="black", label="store_Imm_1_N160_N110"];
N111 [color="black", label="store_Imm_4_N158_N111"];
N112 [color="black", label="store_Imm_6_N159_N112"];
N113 [color="black", label="store_Imm_3_N162_N113"];
N107 [color="black", label="mul_Imm_10_N52_N107"];
N106 [color="black", label="mul_N55_N106"];
N105 [color="black", label="mul_N54_N105"];
N104 [color="black", label="mul_N57_N104"];
N103 [color="black", label="mul_N56_N103"];
N102 [color="black", label="mul_N59_N102"];
N101 [color="black", label="N58N119_mul_N58_add_N119_N101"];
N100 [color="black", label="N146N31_add_N146_add_N31_N100"];
N109 [color="black", label="store_Imm_5_N161_N109"];
N108 [color="black", label="N53N83_mul_Imm_9_N53_add_N83_N108"];
N12 [color="black", label="load_Imm_6_N26_N12"];
N13 [color="black", label="load_Imm_0_N25_N13"];
N10 [color="black", label="load_Imm_8_N20_N10"];
N11 [color="black", label="load_Imm_6_N27_N11"];
N16 [color="black", label="load_Imm_2_N13_N16"];
N17 [color="black", label="load_Imm_3_N10_N17"];
N14 [color="black", label="load_Imm_6_N24_N14"];
N15 [color="black", label="load_Imm_0_N12_N15"];
N18 [color="black", label="load_Imm_3_N11_N18"];
N19 [color="black", label="load_Imm_2_N16_N19"];
N67 [color="black", label="N139N63_mul_N139_add_N63_N67"];
N66 [color="black", label="mul_N135_N66"];
N65 [color="black", label="N134N127_mul_N134_add_N127_N65"];
N64 [color="black", label="mul_N137_N64"];
N63 [color="black", label="N131N99_add_N131_add_N99_N63"];
N62 [color="black", label="mul_Imm_10_N133_N62"];
N61 [color="black", label="N108N35_mul_N108_add_N35_N61"];
N60 [color="black", label="mul_N109_N60"];
N69 [color="black", label="N66N148_mul_N66_add_N148_N69"];
N68 [color="black", label="N67N30_mul_Imm_9_N67_add_N30_N68"];
N8 [color="black", label="load_Imm_4_N22_N8"];
N9 [color="black", label="load_Imm_4_N21_N9"];
N1 [color="black", label="load_Imm_4_N1_N1"];
N2 [color="black", label="load_Imm_7_N2_N2"];
N3 [color="black", label="load_Imm_0_N3_N3"];
N4 [color="black", label="load_Imm_7_N4_N4"];
N5 [color="black", label="load_Imm_8_N5_N5"];
N6 [color="black", label="load_Imm_3_N6_N6"];
N7 [color="black", label="load_Imm_7_N23_N7"];
N74 [color="black", label="N69N128_add_N69_add_N128_N74"];
N75 [color="black", label="mul_N68_N75"];
N76 [color="black", label="N129N145_mul_Imm_9_N129_add_N145_N76"];
N77 [color="black", label="N125N130_mul_Imm_9_N125_add_N130_N77"];
N70 [color="black", label="N65N144_mul_N65_add_N144_N70"];
N71 [color="black", label="N62N28_mul_N62_add_N28_N71"];
N72 [color="black", label="mul_N61_N72"];
N73 [color="black", label="N60N44_mul_N60_add_N44_N73"];
N78 [color="black", label="mul_N124_N78"];
N79 [color="black", label="N126N151_mul_Imm_9_N126_add_N151_N79"];
N49 [color="black", label="N110N147_mul_Imm_9_N110_add_N147_N49"];
N48 [color="black", label="mul_N117_N48"];
N41 [color="black", label="N96N121_mul_N96_add_N121_N41"];
N40 [color="black", label="mul_N91_N40"];
N43 [color="black", label="N94N50_mul_N94_add_N50_N43"];
N42 [color="black", label="N97N51_mul_N97_add_N51_N42"];
N45 [color="black", label="N114N84_mul_N114_add_N84_N45"];
N44 [color="black", label="N95N98_mul_Imm_9_N95_add_N98_N44"];
N47 [color="black", label="N116N138_mul_N116_add_N138_N47"];
N46 [color="black", label="mul_N115_N46"];
N58 [color="black", label="N102N152_mul_N102_add_N152_N58"];
N59 [color="black", label="mul_N100_N59"];
N56 [color="black", label="N104N49_add_N104_add_N49_N56"];
N57 [color="black", label="mul_Imm_10_N103_N57"];
N54 [color="black", label="N107N32_mul_N107_add_N32_N54"];
N55 [color="black", label="N106N74_mul_N106_add_N74_N55"];
N52 [color="black", label="mul_N113_N52"];
N53 [color="black", label="N118N122_add_N118_add_N122_N53"];
N50 [color="black", label="N111N64_mul_N111_add_N64_N50"];
N51 [color="black", label="mul_N112_N51"];
N29 -> N110;
N28 -> N30;
N23 -> N69;
N23 -> N51;
N23 -> N94;
N23 -> N70;
N23 -> N38;
N23 -> N31;
N22 -> N65;
N22 -> N60;
N22 -> N55;
N22 -> N71;
N22 -> N47;
N22 -> N78;
N21 -> N42;
N21 -> N55;
N21 -> N97;
N21 -> N105;
N21 -> N65;
N21 -> N86;
N20 -> N54;
N20 -> N89;
N20 -> N101;
N20 -> N38;
N20 -> N51;
N20 -> N33;
N27 -> N101;
N27 -> N61;
N27 -> N41;
N27 -> N43;
N27 -> N89;
N27 -> N34;
N26 -> N99;
N25 -> N80;
N24 -> N39;
N24 -> N61;
N24 -> N94;
N24 -> N41;
N24 -> N66;
N24 -> N31;
N85 -> N43;
N84 -> N68;
N87 -> N114;
N86 -> N41;
N81 -> N87;
N80 -> N29;
N83 -> N108;
N82 -> N44;
N89 -> N53;
N88 -> N53;
N38 -> N56;
N39 -> N32;
N30 -> N49;
N31 -> N37;
N32 -> N74;
N33 -> N63;
N34 -> N28;
N35 -> N30;
N36 -> N76;
N37 -> N79;
N98 -> N97;
N99 -> N49;
N92 -> N88;
N93 -> N89;
N90 -> N35;
N91 -> N108;
N96 -> N61;
N97 -> N36;
N94 -> N91;
N95 -> N73;
N107 -> N76;
N106 -> N31;
N105 -> N94;
N104 -> N67;
N103 -> N70;
N102 -> N101;
N101 -> N53;
N100 -> N68;
N108 -> N116;
N12 -> N67;
N12 -> N28;
N12 -> N73;
N12 -> N103;
N12 -> N46;
N12 -> N64;
N13 -> N40;
N13 -> N98;
N13 -> N59;
N13 -> N64;
N13 -> N73;
N13 -> N32;
N10 -> N69;
N10 -> N43;
N10 -> N95;
N10 -> N58;
N10 -> N34;
N10 -> N70;
N11 -> N35;
N11 -> N103;
N11 -> N67;
N11 -> N72;
N11 -> N106;
N11 -> N52;
N16 -> N83;
N17 -> N52;
N17 -> N98;
N17 -> N88;
N17 -> N35;
N17 -> N93;
N17 -> N40;
N14 -> N62;
N15 -> N106;
N15 -> N96;
N15 -> N48;
N15 -> N72;
N15 -> N59;
N15 -> N32;
N18 -> N96;
N18 -> N28;
N18 -> N46;
N18 -> N88;
N18 -> N48;
N18 -> N93;
N19 -> N39;
N19 -> N95;
N19 -> N54;
N19 -> N33;
N19 -> N66;
N19 -> N58;
N67 -> N100;
N66 -> N65;
N65 -> N74;
N64 -> N42;
N63 -> N29;
N62 -> N79;
N61 -> N36;
N60 -> N54;
N69 -> N100;
N68 -> N117;
N8 -> N92;
N8 -> N45;
N8 -> N102;
N8 -> N47;
N8 -> N60;
N8 -> N50;
N9 -> N92;
N9 -> N85;
N9 -> N97;
N9 -> N86;
N9 -> N90;
N9 -> N102;
N1 -> N82;
N2 -> N81;
N3 -> N57;
N4 -> N78;
N4 -> N71;
N4 -> N90;
N4 -> N85;
N4 -> N75;
N4 -> N104;
N5 -> N84;
N6 -> N107;
N7 -> N45;
N7 -> N42;
N7 -> N75;
N7 -> N50;
N7 -> N104;
N7 -> N105;
N74 -> N77;
N75 -> N69;
N76 -> N113;
N77 -> N115;
N70 -> N100;
N71 -> N91;
N72 -> N71;
N73 -> N91;
N78 -> N58;
N79 -> N112;
N49 -> N109;
N48 -> N47;
N41 -> N63;
N40 -> N33;
N43 -> N56;
N42 -> N37;
N45 -> N56;
N44 -> N111;
N47 -> N63;
N46 -> N45;
N58 -> N37;
N59 -> N55;
N56 -> N87;
N57 -> N77;
N54 -> N36;
N55 -> N74;
N52 -> N38;
N53 -> N44;
N50 -> N30;
N51 -> N50;
}
