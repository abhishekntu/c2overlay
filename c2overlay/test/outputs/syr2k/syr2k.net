.input N23
pinlist: N23
    
.input N22
pinlist: N22
    
.input N21
pinlist: N21
    
.input N20
pinlist: N20
    
.input N27
pinlist: N27
    
.input N26
pinlist: N26
    
.input N25
pinlist: N25
    
.input N24
pinlist: N24
    
.input N12
pinlist: N12
    
.input N13
pinlist: N13
    
.input N10
pinlist: N10
    
.input N11
pinlist: N11
    
.input N16
pinlist: N16
    
.input N17
pinlist: N17
    
.input N14
pinlist: N14
    
.input N15
pinlist: N15
    
.input N18
pinlist: N18
    
.input N19
pinlist: N19
    
.input N8
pinlist: N8
    
.input N9
pinlist: N9
    
.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.input N4
pinlist: N4
    
.input N5
pinlist: N5
    
.input N6
pinlist: N6
    
.input N7
pinlist: N7
    
.output out:N114
pinlist: N114
    
.output out:N115
pinlist: N115
    
.output out:N116
pinlist: N116
    
.output out:N117
pinlist: N117
    
.output out:N110
pinlist: N110
    
.output out:N111
pinlist: N111
    
.output out:N112
pinlist: N112
    
.output out:N113
pinlist: N113
    
.output out:N109
pinlist: N109
    
.clb N29_blk
pinlist: N63 N80 open open N110 open open open open 
subblock: N29_blk 0 1 open open 4 open open open open

.clb N28_blk
pinlist: N12 N34 N18 open N28 open open open open 
subblock: N28_blk 0 1 2 open 4 open open open open
    
.clb N85_blk
pinlist: N4 N9 open open N85 open open open open 
subblock: N85_blk 0 1 open open 4 open open open open

.clb N84_blk
pinlist: N5 open open open N84 open open open open 
subblock: N84_blk 0 open open open 4 open open open open

.clb N87_blk
pinlist: N56 N81 open open N114 open open open open 
subblock: N87_blk 0 1 open open 4 open open open open

.clb N86_blk
pinlist: N9 N21 open open N86 open open open open 
subblock: N86_blk 0 1 open open 4 open open open open

.clb N81_blk
pinlist: N2 open open open N81 open open open open 
subblock: N81_blk 0 open open open 4 open open open open

.clb N80_blk
pinlist: N25 open open open N80 open open open open 
subblock: N80_blk 0 open open open 4 open open open open

.clb N83_blk
pinlist: N16 open open open N83 open open open open 
subblock: N83_blk 0 open open open 4 open open open open

.clb N82_blk
pinlist: N1 open open open N82 open open open open 
subblock: N82_blk 0 open open open 4 open open open open

.clb N89_blk
pinlist: N20 N27 N93 open N89 open open open open 
subblock: N89_blk 0 1 2 open 4 open open open open
    
.clb N88_blk
pinlist: N17 N18 N92 open N88 open open open open 
subblock: N88_blk 0 1 2 open 4 open open open open
    
.clb N38_blk
pinlist: N52 N20 N23 open N38 open open open open 
subblock: N38_blk 0 1 2 open 4 open open open open
    
.clb N39_blk
pinlist: N19 N24 open open N39 open open open open 
subblock: N39_blk 0 1 open open 4 open open open open

.clb N30_blk
pinlist: N28 N35 N50 open N30 open open open open 
subblock: N30_blk 0 1 2 open 4 open open open open
    
.clb N31_blk
pinlist: N23 N24 N106 open N31 open open open open 
subblock: N31_blk 0 1 2 open 4 open open open open
    
.clb N32_blk
pinlist: N13 N39 N15 open N32 open open open open 
subblock: N32_blk 0 1 2 open 4 open open open open
    
.clb N33_blk
pinlist: N40 N19 N20 open N33 open open open open 
subblock: N33_blk 0 1 2 open 4 open open open open
    
.clb N34_blk
pinlist: N10 N27 open open N34 open open open open 
subblock: N34_blk 0 1 open open 4 open open open open

.clb N35_blk
pinlist: N11 N17 N90 open N35 open open open open 
subblock: N35_blk 0 1 2 open 4 open open open open
    
.clb N36_blk
pinlist: N54 N61 N97 open N36 open open open open 
subblock: N36_blk 0 1 2 open 4 open open open open
    
.clb N37_blk
pinlist: N31 N42 N58 open N37 open open open open 
subblock: N37_blk 0 1 2 open 4 open open open open
    
.clb N98_blk
pinlist: N13 N17 open open N98 open open open open 
subblock: N98_blk 0 1 open open 4 open open open open

.clb N99_blk
pinlist: N26 open open open N99 open open open open 
subblock: N99_blk 0 open open open 4 open open open open

.clb N92_blk
pinlist: N8 N9 open open N92 open open open open 
subblock: N92_blk 0 1 open open 4 open open open open

.clb N93_blk
pinlist: N17 N18 open open N93 open open open open 
subblock: N93_blk 0 1 open open 4 open open open open

.clb N90_blk
pinlist: N4 N9 open open N90 open open open open 
subblock: N90_blk 0 1 open open 4 open open open open

.clb N91_blk
pinlist: N71 N73 N94 open N91 open open open open 
subblock: N91_blk 0 1 2 open 4 open open open open
    
.clb N96_blk
pinlist: N15 N18 open open N96 open open open open 
subblock: N96_blk 0 1 open open 4 open open open open

.clb N97_blk
pinlist: N9 N21 N98 open N97 open open open open 
subblock: N97_blk 0 1 2 open 4 open open open open
    
.clb N94_blk
pinlist: N23 N24 N105 open N94 open open open open 
subblock: N94_blk 0 1 2 open 4 open open open open
    
.clb N95_blk
pinlist: N10 N19 open open N95 open open open open 
subblock: N95_blk 0 1 open open 4 open open open open

.clb N107_blk
pinlist: N6 open open open N107 open open open open 
subblock: N107_blk 0 open open open 4 open open open open

.clb N106_blk
pinlist: N11 N15 open open N106 open open open open 
subblock: N106_blk 0 1 open open 4 open open open open

.clb N105_blk
pinlist: N7 N21 open open N105 open open open open 
subblock: N105_blk 0 1 open open 4 open open open open

.clb N104_blk
pinlist: N4 N7 open open N104 open open open open 
subblock: N104_blk 0 1 open open 4 open open open open

.clb N103_blk
pinlist: N11 N12 open open N103 open open open open 
subblock: N103_blk 0 1 open open 4 open open open open

.clb N102_blk
pinlist: N8 N9 open open N102 open open open open 
subblock: N102_blk 0 1 open open 4 open open open open

.clb N101_blk
pinlist: N20 N27 N102 open N101 open open open open 
subblock: N101_blk 0 1 2 open 4 open open open open
    
.clb N100_blk
pinlist: N67 N69 N70 open N100 open open open open 
subblock: N100_blk 0 1 2 open 4 open open open open
    
.clb N108_blk
pinlist: N83 N91 open open N116 open open open open 
subblock: N108_blk 0 1 open open 4 open open open open

.clb N67_blk
pinlist: N11 N12 N104 open N67 open open open open 
subblock: N67_blk 0 1 2 open 4 open open open open
    
.clb N66_blk
pinlist: N19 N24 open open N66 open open open open 
subblock: N66_blk 0 1 open open 4 open open open open

.clb N65_blk
pinlist: N21 N22 N66 open N65 open open open open 
subblock: N65_blk 0 1 2 open 4 open open open open
    
.clb N64_blk
pinlist: N12 N13 open open N64 open open open open 
subblock: N64_blk 0 1 open open 4 open open open open

.clb N63_blk
pinlist: N33 N41 N47 open N63 open open open open 
subblock: N63_blk 0 1 2 open 4 open open open open
    
.clb N62_blk
pinlist: N14 open open open N62 open open open open 
subblock: N62_blk 0 open open open 4 open open open open

.clb N61_blk
pinlist: N24 N27 N96 open N61 open open open open 
subblock: N61_blk 0 1 2 open 4 open open open open
    
.clb N60_blk
pinlist: N8 N22 open open N60 open open open open 
subblock: N60_blk 0 1 open open 4 open open open open

.clb N69_blk
pinlist: N10 N23 N75 open N69 open open open open 
subblock: N69_blk 0 1 2 open 4 open open open open
    
.clb N68_blk
pinlist: N84 N100 open open N117 open open open open 
subblock: N68_blk 0 1 open open 4 open open open open

.clb N74_blk
pinlist: N32 N55 N65 open N74 open open open open 
subblock: N74_blk 0 1 2 open 4 open open open open
    
.clb N75_blk
pinlist: N4 N7 open open N75 open open open open 
subblock: N75_blk 0 1 open open 4 open open open open

.clb N76_blk
pinlist: N36 N107 open open N113 open open open open 
subblock: N76_blk 0 1 open open 4 open open open open

.clb N77_blk
pinlist: N57 N74 open open N115 open open open open 
subblock: N77_blk 0 1 open open 4 open open open open

.clb N70_blk
pinlist: N10 N23 N103 open N70 open open open open 
subblock: N70_blk 0 1 2 open 4 open open open open
    
.clb N71_blk
pinlist: N4 N22 N72 open N71 open open open open 
subblock: N71_blk 0 1 2 open 4 open open open open
    
.clb N72_blk
pinlist: N11 N15 open open N72 open open open open 
subblock: N72_blk 0 1 open open 4 open open open open

.clb N73_blk
pinlist: N12 N13 N95 open N73 open open open open 
subblock: N73_blk 0 1 2 open 4 open open open open
    
.clb N78_blk
pinlist: N4 N22 open open N78 open open open open 
subblock: N78_blk 0 1 open open 4 open open open open

.clb N79_blk
pinlist: N37 N62 open open N112 open open open open 
subblock: N79_blk 0 1 open open 4 open open open open

.clb N49_blk
pinlist: N30 N99 open open N109 open open open open 
subblock: N49_blk 0 1 open open 4 open open open open

.clb N48_blk
pinlist: N15 N18 open open N48 open open open open 
subblock: N48_blk 0 1 open open 4 open open open open

.clb N41_blk
pinlist: N24 N86 N27 open N41 open open open open 
subblock: N41_blk 0 1 2 open 4 open open open open
    
.clb N40_blk
pinlist: N13 N17 open open N40 open open open open 
subblock: N40_blk 0 1 open open 4 open open open open

.clb N43_blk
pinlist: N10 N85 N27 open N43 open open open open 
subblock: N43_blk 0 1 2 open 4 open open open open
    
.clb N42_blk
pinlist: N7 N21 N64 open N42 open open open open 
subblock: N42_blk 0 1 2 open 4 open open open open
    
.clb N45_blk
pinlist: N7 N8 N46 open N45 open open open open 
subblock: N45_blk 0 1 2 open 4 open open open open
    
.clb N44_blk
pinlist: N53 N82 open open N111 open open open open 
subblock: N44_blk 0 1 open open 4 open open open open

.clb N47_blk
pinlist: N8 N48 N22 open N47 open open open open 
subblock: N47_blk 0 1 2 open 4 open open open open
    
.clb N46_blk
pinlist: N12 N18 open open N46 open open open open 
subblock: N46_blk 0 1 open open 4 open open open open

.clb N58_blk
pinlist: N10 N19 N78 open N58 open open open open 
subblock: N58_blk 0 1 2 open 4 open open open open
    
.clb N59_blk
pinlist: N13 N15 open open N59 open open open open 
subblock: N59_blk 0 1 open open 4 open open open open

.clb N56_blk
pinlist: N38 N43 N45 open N56 open open open open 
subblock: N56_blk 0 1 2 open 4 open open open open
    
.clb N57_blk
pinlist: N3 open open open N57 open open open open 
subblock: N57_blk 0 open open open 4 open open open open

.clb N54_blk
pinlist: N60 N19 N20 open N54 open open open open 
subblock: N54_blk 0 1 2 open 4 open open open open
    
.clb N55_blk
pinlist: N59 N21 N22 open N55 open open open open 
subblock: N55_blk 0 1 2 open 4 open open open open
    
.clb N52_blk
pinlist: N11 N17 open open N52 open open open open 
subblock: N52_blk 0 1 open open 4 open open open open

.clb N53_blk
pinlist: N88 N89 N101 open N53 open open open open 
subblock: N53_blk 0 1 2 open 4 open open open open
    
.clb N50_blk
pinlist: N7 N8 N51 open N50 open open open open 
subblock: N50_blk 0 1 2 open 4 open open open open
    
.clb N51_blk
pinlist: N20 N23 open open N51 open open open open 
subblock: N51_blk 0 1 open open 4 open open open open

