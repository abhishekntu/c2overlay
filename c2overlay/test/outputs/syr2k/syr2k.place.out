Netlist file: test.net   Architecture file: test.arch.xml
Array size: 9 x 9 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N23		7	0	0	#0
N22		0	6	0	#1
N21		2	10	0	#2
N20		10	2	0	#3
N27		10	8	0	#4
N26		10	6	0	#5
N25		7	10	0	#6
N24		0	4	0	#7
N12		3	0	0	#8
N13		0	5	0	#9
N10		4	0	0	#10
N11		0	2	0	#11
N16		6	0	0	#12
N17		10	4	0	#13
N14		1	0	0	#14
N15		0	7	0	#15
N18		10	7	0	#16
N19		0	3	0	#17
N8		10	3	0	#18
N9		10	9	0	#19
N1		9	10	0	#20
N2		9	0	0	#21
N3		0	9	0	#22
N4		2	0	0	#23
N5		1	10	0	#24
N6		4	10	0	#25
N7		10	1	0	#26
out:N114	8	0	0	#27
out:N115	0	8	0	#28
out:N116	5	0	0	#29
out:N117	3	10	0	#30
out:N110	6	10	0	#31
out:N111	8	10	0	#32
out:N112	0	1	0	#33
out:N113	5	10	0	#34
out:N109	10	5	0	#35
N29_blk		6	9	0	#36
N28_blk		6	4	0	#37
N85_blk		7	1	0	#38
N84_blk		2	9	0	#39
N87_blk		8	1	0	#40
N86_blk		5	8	0	#41
N81_blk		9	1	0	#42
N80_blk		7	9	0	#43
N83_blk		6	1	0	#44
N82_blk		9	9	0	#45
N89_blk		7	8	0	#46
N88_blk		8	7	0	#47
N38_blk		8	3	0	#48
N39_blk		1	6	0	#49
N30_blk		9	4	0	#50
N31_blk		1	4	0	#51
N32_blk		2	6	0	#52
N33_blk		6	7	0	#53
N34_blk		6	2	0	#54
N35_blk		8	4	0	#55
N36_blk		5	6	0	#56
N37_blk		1	3	0	#57
N98_blk		7	5	0	#58
N99_blk		9	6	0	#59
N92_blk		8	6	0	#60
N93_blk		7	6	0	#61
N90_blk		7	4	0	#62
N91_blk		4	3	0	#63
N96_blk		3	6	0	#64
N97_blk		5	5	0	#65
N94_blk		4	4	0	#66
N95_blk		5	3	0	#67
N107_blk	4	9	0	#68
N106_blk	1	5	0	#69
N105_blk	4	5	0	#70
N104_blk	2	1	0	#71
N103_blk	5	2	0	#72
N102_blk	9	7	0	#73
N101_blk	9	8	0	#74
N100_blk	3	2	0	#75
N108_blk	5	1	0	#76
N67_blk		2	2	0	#77
N66_blk		1	7	0	#78
N65_blk		2	7	0	#79
N64_blk		2	4	0	#80
N63_blk		6	8	0	#81
N62_blk		1	1	0	#82
N61_blk		4	6	0	#83
N60_blk		6	5	0	#84
N69_blk		3	1	0	#85
N68_blk		3	9	0	#86
N74_blk		2	8	0	#87
N75_blk		4	1	0	#88
N76_blk		5	9	0	#89
N77_blk		1	8	0	#90
N70_blk		4	2	0	#91
N71_blk		3	4	0	#92
N72_blk		3	5	0	#93
N73_blk		5	4	0	#94
N78_blk		2	3	0	#95
N79_blk		1	2	0	#96
N49_blk		9	5	0	#97
N48_blk		4	7	0	#98
N41_blk		4	8	0	#99
N40_blk		7	7	0	#100
N43_blk		7	2	0	#101
N42_blk		2	5	0	#102
N45_blk		7	3	0	#103
N44_blk		8	9	0	#104
N47_blk		5	7	0	#105
N46_blk		6	3	0	#106
N58_blk		3	3	0	#107
N59_blk		3	7	0	#108
N56_blk		8	2	0	#109
N57_blk		1	9	0	#110
N54_blk		6	6	0	#111
N55_blk		3	8	0	#112
N52_blk		8	5	0	#113
N53_blk		8	8	0	#114
N50_blk		9	3	0	#115
N51_blk		9	2	0	#116
